src/Pages/Shared/index.ts

```diff
@@ -1 +1,19 @@
+/*
+ *   Copyright (c) 2024 Devtron Inc.
+ *   All rights reserved.
+
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+
+ *   http://www.apache.org/licenses/LICENSE-2.0
+
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
```

Comment:
The formatting for the copyright is not proper. Please remove the copyright text altogether. I will add to all the files sometime later